toolchain/driver/driver.cpp

```diff
@@ -98,12 +95,8 @@ auto Driver::RunCommand(llvm::ArrayRef<llvm::StringRef> args) -> RunResult {
     driver_env_.vlog_stream = &driver_env_.error_stream;
   }
 
-  switch (options.subcommand) {
-    case Options::Subcommand::Compile:
-      return Compile(options.compile_options, options.codegen_options);
-    case Options::Subcommand::Link:
-      return Link(options.link_options, options.codegen_options);
-  }
+  options.subcommand->Run(driver_env_);
+
   llvm_unreachable("All subcommands handled!");
```

Comment:
I think this should be:

```suggestion
  return options.subcommand->Run(driver_env_);
```

Which should fix the test failue?